llama-index-networks/examples/demo/contributor-1/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.poetry]
+authors = ["Your Name <you@example.com>"]
+description = ""
+name = "contributor-1"
+readme = "README.md"
```

Comment:
If there is no readme, we can remove this line to avoid poetry warnings

Reply:
good call!